src/chrome/content/rules/Turn.com.xml

```diff
@@ -1,97 +1,81 @@
-
 <!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://analytics.turn.com/ => https://analytics.turn.com/: (60, 'SSL certificate problem: self signed certificate')
-
-	CDN buckets:
-
-		- trust.turn.com.edgesuite.net
-		- www.turn.com.edgesuite.net
-
-
-	Nonfunctional subdomains:
-
-		- static *
-
-	* Shows www
-
-
-	Problematic subdomains:
-
-		- ^ ¹
-		- e *
-		- trust ²
-
-	¹ Fastly
-	* Eloqua
-	² Works, akamai
-
-
-	Partially covered subdomains:
-
-		- turn *	(→ a248.e.akamai.net)
-
-	* Avoiding user-visible paths
-
-
-	Fully covered subdomains:
-
-		- (www.)?	(^ → www)
-		- analytics
-		- akamai
-		- analytics
-		- console
-		- d
-		- r
-
-
-	Insecure cookies are set for these domains:
-
-		- .turn.com
-
-
-	Mixed content:
-
-		- Images on console, www from static ¹
-
-		- Web bugs, on www from:
-
-			- d ²
-			- t3.trackalyzer.com ²
-
-	¹ Unsecurable <= 404
-	² Secured by us
+	Cert expired:
+		- autodiscover.turn.com
+		- d-audienceiq-atl1.turn.com
+		- webmail.turn.com
+
+	Chain issues:
```

Comment:
It seems that some of them have been fixed.

Reply:
ping @numismatika

Reply:
Should be correct now